app/src/main/java/com/example/househomey/form/EditItemFragment.java

```diff
@@ -62,6 +67,20 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         return rootView;
     }
 
+    /**
+     * Creates the new Item and waits for tags to finish loading
+     * @param itemId Id of the new item
+     * @param data   Data for the new item
+     * @return New Item with tag listener set
+     */
+    @Override
+    protected Item createItem(String itemId, Map<String, Object> data) {
```

Comment:
Discussed this with @ldbonkowski during my review, we think we can just pass the old item's tags to the new item, to avoid having to wait for the tags to load here.

Using the old tags is safe here because the user can't modify tags in the Add/Edit Item pages.

This would allow us to remove the `createItem` method from the parent/children